examples/rest-xml-json/README_es.adoc

```diff
@@ -0,0 +1,404 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
+= REST Simple 
+
+Definir un servicio REST es bastante fácil, simplemente agregue la anotación @Path a una
+clase luego defina en los métodos el método HTTP a usar (@GET, @POST, …).
+
+El Código
+
+== El servicio REST: @Path, @Produces, @Consumes
+
+Aquí vemos un bean que usa las anotaciones @Path, @Produces y @Consumes.
+La anotación @Path identifica la plantilla de ruta de URI a la que responde el recurso y se especifica en el nivel de clase de un recurso.
+El valor de la anotación @Path
+es una plantilla de ruta de URI parcial relativa al URI base del servidor en el que se implementa el recurso, la raíz de contexto del WAR y el patrón de URL al que
+responde el servlet auxiliar de Jersey.
+La anotación @Produces se utiliza para especificar los tipos de medios MIME o las representaciones que un recurso puede producir y enviar al cliente.
+Si @Produces se aplica en
+el nivel de clase, todos los métodos de un recurso pueden producir los tipos MIME especificados de forma predeterminada. Si se aplica a nivel de método, sobrescribir cualquier @Produces
```

Comment:
"Si se aplica a nivel de método, sobrescribir cualquier @Produces" puede cambiar a "Si se aplica a nivel de método, sobrescribe cualquier @Produces"

Reply:
The correction it is maked.